tests/test_files/filters/test_filter_1.py

```diff
@@ -0,0 +1,15 @@
+from cfripper.config.filter import Filter
+from cfripper.model.enums import RuleMode
+
+FILTERS = [
+    Filter(
+        rule_mode=RuleMode.WHITELISTED,
+        eval={
+            "and": [
+                {"eq": [{"ref": "config.stack_name"}, "mockstack"]},
+                {"eq": [{"ref": "logical_id"}, "RootRoleOne"]},
+            ]
+        },
+        rules={"CrossAccountTrustRule"},
```

Comment:
Is this test case used somewhere? Can't find another reference to `test_filter_1` 😛 

Also, wondering if this test case can be expanded to show a filter being applied on more than one rule (just adding another rule here should be enough).

Reply:
It is used in a couple of places, one is here: https://github.com/Skyscanner/cfripper/blob/d2e0c63a029fa0056f771cc96c10249b10a15a04/tests/rules/test_CrossAccountTrustRule.py#L142

I reference the directory to load the filters and not the file itself.
I will add that test, I agree it is needed